src/db.ts

```diff
@@ -300,11 +299,11 @@ export abstract class Database {
     },
 
     fetchMaxMeasurements: `SELECT
-        runId, criterion, invocation as inv, max(iteration) as ite
+        runId, criterion, invocation as inv
       FROM Measurement
       WHERE trialId = $1
       GROUP BY runId, criterion, invocation
-      ORDER BY runId, inv, ite, criterion`,
```

Comment:
This looks wrong.

The query is called `fetchMaxMeasurements`, so, I suspect the whole point is to get the number of measurements, no?

how about `cardinality(value) as ite` here?